Parse/Internal/LocalDataStore/SQLite/PFSQLiteDatabaseResult.m

```diff
@@ -12,11 +12,20 @@
 #import <sqlite3.h>
 
 #import "PFSQLiteStatement.h"
+#import "PFThreadsafety.h"
+
+// PFThreadsafetySafeDispatchSync, but with a return type.
+#define thread_safe_sync(queue, block) ({                          \
```

Comment:
I'm not sure how we can, seeing as the point of this macro is to be able to easily return any type, not just `id` or `int`.

We could stick it in `PFThreadsafety.h`, perhaps?